test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -10929,6 +10886,48 @@ def log_match(self, msg=None, id=None, n=50, tail=True, allmatch=False,
                                max_attempts, interval, starttime, endtime,
                                level=level, existence=existence)
 
+    def run_scheduling_cycle(self):
+        """
+        Convenience method to start and finish a sched cycle
+
+        :return start time of the new sched cycle
+        """
+        sched = self.attributes['id']
+        old_val = self.server.status(SCHED, 'scheduling', id=sched)[
+            0]['scheduling']
+
+        # Make sure that we aren't in a sched cycle already
+        self.server.manager(MGR_CMD_SET, SCHED, {
+                            'scheduling': 'False'}, id=sched)
+
+        # Kick a new cycle
+        tbefore = time.time()
+        self.server.manager(MGR_CMD_SET, SCHED, {
+                            'scheduling': 'True'}, id=sched)
+        msg = self.log_match("Starting Scheduling",
+                             starttime=tbefore)
+        self.log_match("Leaving Scheduling",
+                       starttime=tbefore, interval=1,
+                       max_attempts=1200)
+
+        # Get start timestamp of the sched cycle
+        if self.logutils is None:
+            try:
+                from ptl.utils.pbs_logutils import PBSLogUtils
+            except:
+                _msg = 'error loading ptl.utils.pbs_logutils'
+                raise PtlLogMatchError(rc=1, rv=False, msg=_msg)
```

Comment:
This isn't really a log match error, is there a better exception to raise?  A more generic log utils one?

Reply:
you are right, i think i copied this from somewhere. This really is just an import error, so I'll raise the same